src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -188,6 +186,26 @@ public LivingEntity(ModElement element) {
 
 		this.inventorySize = 9;
 		this.inventoryStackSize = 64;
+
+		this.modelLayers = new ArrayList<>();
+	}
+
+	public static class ModelLayerEntry implements IWorkspaceDependent {

```

Comment:
It may be worth adding this to the test cases, for texture reference finding (as part of this PR).

> Should be.

It would be good to also test manually with model finding if you have not yet.

Reply:
Done locally, I'm not that fast.

Reply:
You have requested a review and thought you did not see my comment, given you requested a review on a PR that already had my review pending

> I'm not that fast

I never ask anyone to do anything immediately, you are free to do the changes when you have time :)

Reply:
✔️ Fixed